src/timeseriesflattener/feature_spec_objects.py

```diff
@@ -449,7 +450,7 @@ def get_col_str(self, additional_feature_name: Optional[str] = None) -> str:
         feature_name = self.feature_name
         if additional_feature_name:
             feature_name = feature_name + "-" + str(additional_feature_name)
-        col_str = f"{self.prefix}_{feature_name}_within_{self.interval_days}_days_{self.key_for_resolve_multiple}_fallback_{self.fallback}"
+        col_str = f"{self.prefix}_{feature_name}_within_{int(self.interval_days) if self.interval_days.is_integer() else str(self.interval_days).replace('.', '_')}_days_{self.key_for_resolve_multiple}_fallback_{self.fallback}"
```

Comment:
I think this is too much code to have within an `f-string`. Reading it is pretty hard. 

Try extracting `int(self.interval_days) if self.interval_days.is_integer() else str(self.interval_days).replace('.', '_')` into its own variable.

Also, ideally, we want each "unit" of information to be separated by `_`, as that's the current pattern.

Converting `1.3` days to `1_3_days` means that there's now an extra extra `_`, signifying an extra "unit" of information, which breaks some parsing I've written for e.g. `t2d-baseline-paper`. Is it possible to use another separator instead?

Reply:
Agree, with both points in this comment. Thanks for fixing the first point. Regarding another separator, I don't have any particular preferences. Perhaps use `-`?

Reply:
Sounds good 👍